tests/integration/devfile/cmd_delete_test.go

```diff
@@ -252,4 +252,66 @@ ComponentSettings:
 			})
 		})
 	})
+
+	//Test reused and adapted from the now-removed `cmd_devfile_delete_test.go`
+	When("deleting a component that has resources attached to it and was deployed with Dev", func() {
+		resourceTypes := []string{
+			helper.ResourceTypeDeployment,
+			helper.ResourceTypePod,
+			helper.ResourceTypeService,
+			helper.ResourceTypeIngress,
+			helper.ResourceTypePVC,
+		}
+		BeforeEach(func() {
+			// Component name comes from devfile-with-endpoints.yaml
+			cmpName = "nodejs-with-endpoints"
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
+			helper.Cmd("odo", "project", "set", commonVar.Project).ShouldPass()
+			helper.Cmd("odo", "init", "--name", cmpName, "--devfile-path",
+				helper.GetExamplePath("source", "devfiles", "nodejs", "devfile-with-endpoints.yaml")).ShouldPass()
+		})
+		When("a Dev session is started and abruptly killed", func() {
+			BeforeEach(func() {
+				//Purposely killing the dev session to test the component deletion behavior.
+				//Also, devSession.Stop() is expected to clean-up resources at some point
+				helper.StartDevMode().Kill()
+			})
+			It("should delete the component and its owned resources", func() {
```

Comment:
I meant that I am purposely killing the Dev mode, not just stopping it cleanly, because (in my understanding) #5485 should delete all these resources upon a clean exit in a near future.
At least, this is the behavior I deduced from reading the [original test](https://github.com/redhat-developer/odo/blob/24fd02673d25eb4c7bb166ec3369554a8e64b59c/tests/integration/devfile/cmd_devfile_delete_test.go#L172-L238), but I might have missed something ^^ 
But you are right, I will reword this sentence.